assets/js/blocks/price-filter/block.js

```diff
@@ -108,6 +123,33 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 		minorUnit: currency.minorUnit,
 	} );
 
+	/**
+	 * Important: For PHP rendered block templates only.
+	 *
+	 * When we render the PHP block template (e.g. Classic Block) we need
+	 * to set the default min_price and max_price values from the URL
+	 */
```

Comment:
Let's also mention in this comment that we need to set the default state so this filter works alongside the Active Filters block.